primitives/blockchain/src/error.rs

```diff
@@ -35,110 +35,150 @@ pub enum ApplyExtrinsicFailed {
 	/// unappliable onto the current block.
 	#[error("Extrinsic is not valid: {0:?}")]
 	Validity(#[from] TransactionValidityError),
+
 	/// This is used for miscellaneous errors that can be represented by string and not handleable.
 	///
 	/// This will become obsolete with complete migration to v4 APIs.
+	#[deprecated(note = "Introduce more typed error variants as needed")]
```

Comment:
I don't think this is user controllable, also the doc comment says it will become obsolete with v4 APIs.

Reply:
Hmm, thought I already removed the deprecation notice.

Reply:
Is this still used somewhere? I grepped for it but didn't found any usage?

Reply:
Not directly, but via https://github.com/paritytech/substrate/pull/7472/files/2a09d3c9d85ddcc685e754edb00fe6ba9ee4b2fa#diff-7335c8ce9957676e8ab54662cbc046f4a716d3d24f5d2c80faf888e4c4e29806R196-R207